Sming/Makefile-rboot.mk

```diff
@@ -29,7 +29,7 @@ RBOOT_ROM_0      ?= rom0
 RBOOT_ROM_1      ?= rom1
 RBOOT_SPIFFS_0   ?= 0x100000
 RBOOT_SPIFFS_1   ?= 0x300000
-RBOOT_LD_0 ?= $(SMING_HOME)/compiler/ld/rboot.rom0.ld
+RBOOT_LD_0 ?= rboot.rom0.ld
```

Comment:
Will it be possible with those changes to have, if needed, application specific rboot.rom.ld or eagle.app.v6.cpp.include.ld that differs from framework one?